### PR TITLE
[Documentation] Adding a new doc explaining testnet vs devnet vs ait.

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Motivated by this [Aptos-internal-only Slack thread](https://aptos-org.slack.com/archives/C03EG004E56/p1653073695685849), this PR introduces a table explaining the differences between devnet, testnet and ait. 

cc: @gregnazario

### CODE REVIEW

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
+
+You can connect to Aptos Blockchain in a few ways. See the below table:
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:8]: nit: connect to *the* Aptos Blockchain...

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:6]: nit: can we change "Testnet" to "Local Testnet" throughout the document? I think it'll make things much cleaner.

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+:::tip
+
+Make sure to see the row describing **What not to do**.
+
+:::
+
+|Description |Aptos Devnet | Testnet | Aptos Incentivized Testnet (AIT)|
+|---|---|---|---|
+|**Network runs where**| Runs on Aptos servers.| Runs locally on your computer, entirely independent from devnet. | Runs on Aptos servers.|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:18]: This is not quite true:
- Aptos Devnet: Validators run on Aptos servers. Fullnodes are both run by Aptos and the public (i.e., the Aptos community).
- Local Testnet: Validators and Fullnodes run locally on your computer.
- Aptos Incentivized Testnet (AIT): Some Validators run on Aptos servers, others are community run. Fullnodes are both run by Aptos and the public (i.e., the community)

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+|**Who is responsible for the network**|Managed by Aptos Team. | Deployed, managed and upgraded by you.| Managed by Aptos Team.|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:19]: nit: Aptos Incentivized Testnet (AIT) is managed by Aptos and the community.

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+|**Purpose of the network**|The devnet is built for experimenting with new ideas, improve performance and enhance the user experience.| The testnet is for your development purposes and runs on your local computer.| For executing the Aptos Incentivized Testnet programs for the community.|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:20]: nit, I'd say: "The devnet is built to experiment with new ideas, improve performance and enhance the user experience."

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+|**Network status**|Mostly live, with brief interruptions during regular updates. | On your local computer. | **Live only during Incentivized Testnet drives**. |
+|**Type of nodes**|Validator nodes on Aptos servers. | Validator node on your local computer. | AIT FullNode and AIT validator nodes on your local computers.|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:22]: All three deployment modes have validators and fullnodes. See my comment above.

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+|**Purpose of the nodes**|Validator node in the devnet decides if and when your transaction should be added to the Aptos Blockchain.|You can connect your local validator node to your local testnet for development.| Your local AIT FullNode and AIT validator nodes must satisfy the [Deploying for Aptos Incentivized Testnet](validator-node/intro.md#deploying-for-aptos-incentivized-testnet) criteria.
+|**What not to do**| Do not attempt to sync your local AIT FullNode or AIT validator node with devnet. | Do not attempt to sync your local tesnet validator node with Aptos Incentivized Testnet. | Make sure your deploy your local AIT FullNode and AIT validator node in the test mode, and follow the instructions in [Deploying for Aptos Incentivized Testnet](validator-node/intro.md#deploying-for-aptos-incentivized-testnet).|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:25]: nits:
- "tesnet" -> "testnet"
- "Make sure your" -> "Make sure you"

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+|**Type of nodes**|Validator nodes on Aptos servers. | Validator node on your local computer. | AIT FullNode and AIT validator nodes on your local computers.|
+|**How to connect to the network**|<ul><li> You can transact directly with devnet without a local testnet. See, for example, [Your first transaction](/tutorials/first-transaction.md).</li><li> You can run FullNode locally on your computer and synchronize with devnet. See [Run a FullNode](/tutorials/full-node/run-a-fullnode.md).</li></ul>| You can start a validator node locally and connect with your local testnet for development purposes. | You must start both a local AIT FullNode and a local AIT validator node to connect to the AIT-1.|
+|**Purpose of the nodes**|Validator node in the devnet decides if and when your transaction should be added to the Aptos Blockchain.|You can connect your local validator node to your local testnet for development.| Your local AIT FullNode and AIT validator nodes must satisfy the [Deploying for Aptos Incentivized Testnet](validator-node/intro.md#deploying-for-aptos-incentivized-testnet) criteria.
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:24]: nit: I think we can remove this line. All 3 deployment models allow validators and fullnodes. What do we think?

File: developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+|**Purpose of the network**|The devnet is built for experimenting with new ideas, improve performance and enhance the user experience.| The testnet is for your development purposes and runs on your local computer.| For executing the Aptos Incentivized Testnet programs for the community.|
+|**Network status**|Mostly live, with brief interruptions during regular updates. | On your local computer. | **Live only during Incentivized Testnet drives**. |
+|**Type of nodes**|Validator nodes on Aptos servers. | Validator node on your local computer. | AIT FullNode and AIT validator nodes on your local computers.|
+|**How to connect to the network**|<ul><li> You can transact directly with devnet without a local testnet. See, for example, [Your first transaction](/tutorials/first-transaction.md).</li><li> You can run FullNode locally on your computer and synchronize with devnet. See [Run a FullNode](/tutorials/full-node/run-a-fullnode.md).</li></ul>| You can start a validator node locally and connect with your local testnet for development purposes. | You must start both a local AIT FullNode and a local AIT validator node to connect to the AIT-1.|
```
Comment (JoshLind) [inline developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md:23]: nits:
- Devnet: "You can run FullNode locally" -> "You can run FullNodes locally"
- Local Testnet: "You can start a validator node locally..." -> "You can start a validator and Fullnode locally..."
- AIT: "You must start both a local AIT FullNode and a local AIT validator node to connect to the AIT-1." -> "You must start a local AIT validator node to connect to the AIT-1. FullNodes are optional and can also be connected to AIT-1"

**Review by JoshLind — COMMENTED** [review]
Thanks @rajkaramchedu! Looks great 😄 

Added a bunch of comments around technical clarifications. Once we fix these nits, we should be good to go!

**Review by JoshLind — APPROVED** [review]
Awesome, thanks @rajkaramchedu 😄

File: developer-docs-site/docs/tutorials/local-testnet-devnet-incentivized-testnet.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+|**Type of nodes** (also see the row **Network runs where**) |Validators and FullNodes. | Validators and FullNodes. | Validators and FullNodes.|
+|**How to connect to the network**|<ul><li> You can transact directly with devnet without a local testnet. See, for example, [Your first transaction](/tutorials/first-transaction.md).</li><li> You can run FullNodes locally on your computer and synchronize with devnet. See [Run a FullNode](/tutorials/full-node/run-a-fullnode.md).</li></ul>| You can start a Validator and FullNode locally and connect with your local testnet for development purposes. | You must start both a local AIT Validator node locally to connect to the AIT-1. Optionally, FullNodes can also be run locally and connected to AIT-1.|
+|**What not to do**| Do not attempt to sync your local AIT FullNode or AIT Validator node with devnet. | Do not attempt to sync your local testnet Validator node with Aptos Incentivized Testnet. | Make sure you deploy your local AIT FullNode and AIT alidator node in the test mode, and follow the instructions in [Deploying for Aptos Incentivized Testnet](validator-node/intro.md#deploying-for-aptos-incentivized-testnet).|
```
[the remark this answers is not in the document]
Comment (rajkaramchedu) [reply developer-docs-site/docs/tutorials/local-testnet-devnet-incentivized-testnet.md]: Thanks, @JoshLind. Fixt.

File: developer-docs-site/docs/intro.md [hunk 11]
```diff
@@ -10,39 +10,39 @@ hide_table_of_contents: true
 
 Welcome! Aptos is building a Layer 1 for everyone. Here in our documentation you can explore tutorials and technical concepts for the Aptos Network.
 
-## 🤔 Got a question?
+## Want to dive in?
 
-Join our [Discord][discord] to speak with our developers and hop into the Aptos community. It's the best way to keep up to date with news and developments in the Aptos universe. Be sure to check pinned messages in the channels - this is where we like to keep topic-specific links, events, and more.
+- Start with the basics, such as [Transactions and States](bsics/basics-txns-states) and how [Accounts](/basics/basics-accounts) work.
```
Comment (davidiw) [inline developer-docs-site/docs/intro.md:15]: how did this pass netlify checks? @rajkaramchedu 

note: bsics

**Pull request merged**